k8s/namespaces/fees-pay/ccpay-payment-api-int/demo.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-payment-api-int
+spec:
+  releaseName: ccpay-payment-api-int
+  values:
+    java:
+      image: hmctspublic.azurecr.io/payment/api:pr-1202-27bcf84-20220630101545 #{"$imagepolicy": "flux-system:demo-ccpay-payment-app"}
```

Comment:
Current imagepolicy is not reflective for pr-1202

Reply:
Updated image to 1190 which is currently deployed in demo